sdk/core/Azure.Core/tests/public/ModelSerializationTests/ModelXTests.cs

```diff
@@ -37,6 +44,8 @@ protected override void CompareModels(ModelX model, ModelX model2, ModelSerializ
         protected override string GetExpectedResult(ModelSerializerFormat format)
         {
             string expected = "{\"kind\":\"X\",\"name\":\"xmodel\"";
+            expected += ",\"fields\":[\"testField\"]";
```

Comment:
we talked about having a convention for all of these strings.  Something along the lines of

Model/ModelWire.json
Model/ModelJson.json
Model/ExpectedWire.json
Model/ExpectedJson.json

From this convention the ModelTests base class could simply load all of those files and do all comparisons.

Did we decide to punt this?  If so can you open a tracking issue so we don't lose track?

Reply:
I wasn't aware that we had decided on this convention - I made an issue to track the work: https://github.com/Azure/azure-sdk-for-net/issues/38404